functions.qmd

```diff
@@ -166,26 +166,27 @@ rescale01 <- function(x) {
   rng <- range(x, na.rm = TRUE, finite = TRUE)
   (x - rng[1]) / (rng[2] - rng[1])
 }
+
 rescale01(x)
 ```
 
 These changes illustrate an important benefit of functions: because we've moved the repeated code into a function, we only need to make the change in one place.
 
 ### Mutate functions
 
-Now you've got the basic idea of functions, lets take a look a whole bunch of examples.
-We'll start by looking at "mutate" functions, functions that work well like `mutate()` and `filter()` because they return an output the same length as the input.
+Now you've got the basic idea of functions, let's take a look at a whole bunch of examples.
+We'll start by looking at "mutate" functions, i.e. functions that work like `mutate()` and `filter()` because they return an output of the same length as the input.
```

Comment:
Maybe I am not understanding it correctly (it's late already). How do `filter()` and `mutate()` return an output of the same length as the input? Mutate can add new columns to the tibble and filter often reduces the amount of rows.

Reply:
I think this should be "work well _inside_ of `mutate()` and `filter()`"